Havtorn/Source/Engine/Scene/Scene.cpp

```diff
@@ -285,17 +288,26 @@ namespace Havtorn
 
 		//bool useLifeTime = true;
 		//bool ignoreDepth = true;
-		//F32 durationSeconds = 60.0f;
+		//F32 durationSeconds = 10.0f;
 		//F32 thickness = Debug::UDebugShapeSystem::ThicknessMaximum;
 		//Debug::UDebugShapeSystem::AddLine(SVector(0.0f, 0.0f, 0.0f), SVector(1.0f, 0.0f, 0.0f), Color::Red, durationSeconds, useLifeTime, thickness, ignoreDepth);
 		//Debug::UDebugShapeSystem::AddLine(SVector(0.0f, 0.0f, 0.0f), SVector(0.0f, 1.0f, 0.0f), Color::Green, durationSeconds, useLifeTime, thickness, ignoreDepth);
 		//Debug::UDebugShapeSystem::AddLine(SVector(0.0f, 0.0f, 0.0f), SVector(0.0f, 0.0f, 1.0f), Color::Blue, durationSeconds, useLifeTime, thickness,  ignoreDepth);
-
-		//ignoreDepth = false;
-		//thickness = Debug::UDebugShapeSystem::ThicknessMinimum;
-		//Debug::UDebugShapeSystem::AddLine(SVector(0.0f, 0.0f, 0.0f), SVector(-1.0f, 0.0f, 0.0f), Color::Yellow, durationSeconds, useLifeTime, thickness, ignoreDepth);
-		//Debug::UDebugShapeSystem::AddLine(SVector(0.0f, 0.0f, 0.0f), SVector(0.0f, -1.0f, 0.0f), Color::Teal, durationSeconds, useLifeTime, thickness, ignoreDepth);
-		//Debug::UDebugShapeSystem::AddLine(SVector(0.0f, 0.0f, 0.0f), SVector(0.0f, 0.0f, -1.0f), Color::Purple, durationSeconds, useLifeTime, thickness,  ignoreDepth);
+		//
```

Comment:
Don't love the mass of comments, is it possible to put it in a separate function that you can just call when you need it?

Reply:
Commiting this file was a mistake. Will remove it.